src/chains-config/kusamaControllers.ts

```diff
@@ -8,6 +8,7 @@ export const kusamaControllers: ControllerConfig = {
 	controllers: {
 		Blocks: true,
 		BlocksExtrinsics: true,
+		AccountsAssets: true,
```

Comment:
Ideally we should just state which controllers we want and assume that all controllers not explicitly mentioned are not available. So it'd be `controllers = [Blocks, BlocksExtrinsics, AccountAssets, …, …];`. The controllers for kilt, dock, kulupu etc are getting pretty boilerplatey.